app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -311,12 +315,10 @@
                         } else {
                             set_map[tiers_string] = [set];
                         }
-                        vm.allSetTiers = Object.keys(set_map).sort().reverse();
-                        vm.activesets = vm.allSetTiers[0];
                     }
                     
                     processed_count++;
-                    if((processed_count%200) == 0) {
+                    if((processed_count%1000) == 0) {
```

Comment:
What do you all think about this number? In my testing I can crank it up to ~5000 and still get mediocre browser responsiveness, but I'd guess that everyone's machine will handle it differently. 

At the ~5000 number, I was seeing loading times of about 4 seconds on what was about 12.